src/ApiHandler.ts

```diff
@@ -139,6 +125,69 @@ export default class ApiHandler {
 			}
 		}
 
+		const byteFee = api.consts.transactionPayment.transactionByteFee;
+		const extrinsicBaseWeight = api.consts.system.extrinsicBaseWeight;
+		const coefficients = api.consts.transactionPayment.weightToFee[0];
+		const coeffFrac = coefficients.coeff_frac;
+		const coeffInt = coefficients.coeff_integer;
+		
+		// TODO: generalize for terms that may have more degrees & may be negative
+		const degree = coefficients.degree;
+		const negative = coefficients.negative;
+
+		for (let idx = 0; idx < block.extrinsics.length; ++idx) {
```

Comment:
```suggestion
		for (extrinsic record of extrinsics) {
```

Reply:
I still needed access to the "raw" extrinsic from the block in some cases so I was not able to figure out how to adopt this suggestion.

Reply:
the only place I can spot where you use it is where you read the `encodedLenght` of the extrinsic. I am surprised that this is not contained in the result. Maybe just add this. Might be an interesting information anyways.

Reply:
I also use it to get the hex of the extrinsic when I follow the old code path for Kusama backward compatibility.

Reply:
Far enough. Then just leave it the way it is. I am no javascript loop expert :D

Reply:
That makes two of us!